src/node_builtins.cc

```diff
@@ -501,14 +501,16 @@ void BuiltinLoader::CopyCodeCache(std::vector<CodeCacheInfo>* out) const {
 
 void BuiltinLoader::RefreshCodeCache(const std::vector<CodeCacheInfo>& in) {
   RwLock::ScopedLock lock(code_cache_->mutex);
+  code_cache_->map.reserve(in.size());
+  CHECK_EQ(code_cache_->map.size(), 0);
   for (auto const& item : in) {
-    size_t length = item.data.size();
-    uint8_t* buffer = new uint8_t[length];
-    memcpy(buffer, item.data.data(), length);
     auto new_cache = std::make_unique<v8::ScriptCompiler::CachedData>(
-        buffer, length, v8::ScriptCompiler::CachedData::BufferOwned);
+        item.data.data(),
+        item.data.size(),
+        v8::ScriptCompiler::CachedData::BufferNotOwned);
     code_cache_->map[item.id] = std::move(new_cache);
```

Comment:
This can just be:

```cpp
auto result = code_cache_->map.emplace(id, std::make_unique<v8::ScriptCompiler::CachedData>(...));
DCHECK(result.second);
```

Reply:
I had to add `USE(result.second);` to avoid getting a warning in release mode, but otherwise done.